### ISSUE TITLE
Checking STRT & STOP Depths Incorrectly Identifies Non-Conformity

### ISSUE THREAD

**Issue by andymcdgeo** [MandarJKulkarni/lascheck#12]
When using the LAS check on a las file from the Volve Field ( [15_9-F-1A.LAS.zip](https://github.com/MandarJKulkarni/lascheck/files/9555154/15_9-F-1A.LAS.zip) ), the following issues are reported:

```
['STRT divided by step is not a whole number',
 'STOP divided by step is not a whole number',
 'Section ~Version having blank line',
 'Section ~Well having blank line',
 'Section ~Curve having blank line',
 'Section ~Parameter having blank line',
 'Section ~Other having blank line',
 'Section ~Other having blank line']
```

Within the file the start depth is 145.9 and the step is 0.1. When the depth is divided by 0.1 it results in 1459.0. 

However, when doing what the following class does with the modulo operator (`%`), it returns back 0.09999999999999759. This potentially indicates that the floating point values are not being handled correctly

```
class ValidDepthDividedByStep(Rule):
    @staticmethod
    def check(las_file):
        if "Well" in las_file.sections and 'STRT' in las_file.well and \
                'STOP' in las_file.well and 'STEP' in las_file.well:
            las_file.non_conforming_depth = []
            if las_file.well['STRT'].value % las_file.well['STEP'].value != 0:
                las_file.non_conforming_depth.append('STRT')


            if las_file.well['STOP'].value % las_file.well['STEP'].value != 0:
                las_file.non_conforming_depth.append('STOP')
            return las_file.non_conforming_depth.__len__() == 0
        return False
```

One way to solve this would be round to X decimal places, however, I am sure there are much better ways of handling floating point errors.

**Comment by MandarJKulkarni** [comment]
Fixed this with version 0.1.5
